src/main/java/io/botcrafting/botcraft/core/handler/processors/MessageProcessor.java

```diff
@@ -0,0 +1,10 @@
+package io.botcrafting.botcraft.core.handler.processors;
+
+import io.botcrafting.botcraft.core.model.Message;
+import io.botcrafting.botcraft.core.service.BookService;
+import io.botcrafting.botcraft.core.service.MessageSenderService;
+
+public interface MessageProcessor {
+
+	public boolean processMessage(Message message);
```

Comment:
MessageProcessor looks too generics, how is this interface processing messages? For us is clear, but just for the sake of clarity we could give it a more specific name. Suggestion: MessageReplier